crates/matrix-sdk/src/oidc/tests.rs

```diff
@@ -29,22 +31,24 @@ use crate::{
 
 const CLIENT_ID: &str = "test_client_id";
 
-pub fn fake_registered_client_data() -> RegisteredClientData {
-    RegisteredClientData {
-        credentials: ClientCredentials::None { client_id: CLIENT_ID.to_owned() },
-        metadata: ClientMetadata {
-            redirect_uris: Some(vec![]), // empty vector is ok lol
-            token_endpoint_auth_method: Some(OAuthClientAuthenticationMethod::None),
-            ..ClientMetadata::default()
-        }
-        .validate()
-        .expect("validate client metadata"),
+pub fn fake_client_credentials() -> ClientCredentials {
```

Comment:
I know we've used this "fake" naming before, but why not just "mock" here as well?